<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_addTable</class>
 <widget class="QDialog" name="dlg_addTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> QDialog{   background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(240, 245, 235, 1), stop:0.5 rgba(255, 255, 255, 1), stop:1 rgba(240, 245, 235, 1)); /* Gradient from light green to white to light green */
    color: #333333; /* Dark Grey */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
}
QTableWidget {
    background-color: #FFFFFF; /* White */
    color: #333333; /* Dark Grey */
    gridline-color: #CCCCCC; /* Light Grey */
    border: 1px solid #CCCCCC; /* Light Grey Border */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
}
QLineEdit{
background-color: #FFFFFF; /* White */
    color: #333333; /* Dark Grey */
    gridline-color: #CCCCCC; /* Light Grey */
    border: 1px solid #CCCCCC; /* Light Grey Border */
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
}

/* Buttons common style */

QPushButton {
    background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #E0E0E0, stop:1 #FFFFFF); /* Light Grey to White Gradient */
    color: #333333; /* Dark Grey */
    border: 1px solid #CCCCCC; /* Light Grey Border */
    border-radius: 4px;
    padding: 5px;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 10pt;
}

QPushButton:hover {
    background-color: #D6D6D6; /* Slightly Darker Grey */
}

QPushButton:pressed {
    background-color: #BFBFBF; /* Darker Grey */
}
/* Common background widget style */

</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>餐桌编号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>餐桌容量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>餐桌状态</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_table_free">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>空闲</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_table_busy">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>忙碌</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tablecap_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tablecap_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tablecap_8">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
